valhalla/mjolnir/osmway.h

```diff
@@ -2658,6 +2673,7 @@ struct OSMWay {
   uint32_t truck_route_ : 1;
   uint32_t sidewalk_right_ : 1;
   uint32_t sidewalk_left_ : 1;
+  uint32_t multiple_levels_ : 1;
```

Comment:
for this to work you'll have to move this down to where the spare is and subtract one from the spare

Reply:
if you dont it will work but the compiler will do a bunch of padding and waste a ton of ram/disk